app/frontend/wallet/shelley/shelley-transaction.ts

```diff
@@ -117,9 +117,9 @@ function cborizeTxInputs(inputs: TxInput[]): CborizedTxInput[] {
   return txInputs
 }
 
-function cborizeTxOutputTokens(tokens: Token[]): CborizedTxTokens {
+function cborizeTxOutputTokens(tokenBundle: TokenBundle): CborizedTxTokens {
```

Comment:
this begs for renaming to `cborizeTxOutputTokenBundle` and the type would be `CborizedTxTokenBundle` then

Reply:
thnx :+1: